### PR TITLE
define `daml_ledger_export_test` in terms of `client_server_test`

### PR DESCRIPTION
**Pull request by akrmn**
This should prevent these tests from getting stuck forever. On my machine they take about 15s, so if they take longer than `timeout="short"` (1 min, https://bazel.build/reference/be/common-definitions#test.timeout) we can individually increment their timeouts or mark them as flaky.

### CODE REVIEW

File: daml-script/export/integration-tests/golden/util.bzl [hunk 1]
```diff
@@ -20,60 +20,71 @@ def daml_ledger_export_test(
...
-            ],
-            client = "//daml-script/export/integration-tests/example-export-client",
+            client = client_name,
             client_args = [
                 "--target-port=%PORT%",
                 "--script-identifier={}".format(script_identifier),
                 "--party=" + ",".join(parties),
             ],
-            client_files = [dar],
+            client_files = ["$(rootpath {})".format(dar)],
```
Comment (remyhaemmerle-da) [inline daml-script/export/integration-tests/golden/util.bzl:45]:
```suggestion
            client_files = ["$(rootpath %)" % dar],
```

Comment (remyhaemmerle-da) [reply daml-script/export/integration-tests/golden/util.bzl:45]: Personal preference, but I am fine you keep format if you really prefer it.

**Review by remyhaemmerle-da — APPROVED** [review]
Nice.

File: daml-script/export/integration-tests/golden/util.bzl (hunk 1 above)
Comment (akrmn) [reply daml-script/export/integration-tests/golden/util.bzl:45]: ah that works better here, thanks

**Comment by garyverhaegen-da** [comment]
Thanks!

**Pull request merged**